guides/doc-Managing_Content/master.adoc

```diff
@@ -71,7 +67,8 @@ endif::[]
 include::common/modules/proc_importing-content-isos-into-a-connected-foreman.adoc[leveloffset=+1]
 
 endif::[]
-ifndef::satellite[]
+
+ifdef::katello,orcharhino[]
```

Comment:
I've decided to modify this condition, because the content is not relevant for foreman-deb and foreman-el.

Reply:
👍